zerver/webhooks/updown/doc.md

```diff
@@ -1,16 +1,31 @@
+# Zulip Updown integration
+
 See Updown reports in Zulip!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. Go to [your Updown settings page](https://updown.io/settings/edit).
+   Under **Webhooks**, set **Webhook endpoint URLs** to the URL
+   generated above, and click on **+**.
 
-1. Under **Webhooks**, set **Webhook endpoint URLs** to the URL
-   constructed above, and click on **+**.
```

Comment:
I'd leave this as two instructions, but maybe revise to be ...
```
1. On to [your Updown settings page](https://updown.io/settings/edit),
   go to **Webhooks**.
   
1. Add the URL generated above to the list of **Webhook endpoint URLs**.
```

I based that last one on the first line for setting up a webhook in their API docs here: https://updown.io/api#webhooks. The plus alone looks a little strange when rendered.